docs/pull-request-comments.md

```diff
@@ -0,0 +1,29 @@
+# Pull Request Comments
+
+Private Packagist pull request comments help you review dependency changes in your VCS repositories. Every time a
+composer.lock file changes in one of your repositories, we comment on the pull request with a human-readable summary,
+links to diffs and changelogs for you to review. Pull request comments are available for synchronizations with
+GitHub (Enterprise), GitLab and Bitbucket.
+
+We detect two kinds of changes:
+* Packages that are added, removed or where the version changed
+* Packages with important metadata changes, like changes to the source or dist URL or if the reference changes for a tag
```

Comment:
```suggestion
We detect two kinds of changes:
* Package changes: Additions, removals, upgrades, and downgrades
* Important metadata changes: Changes to the source or dist download URLs or if the commit reference changes for an otherwise unmodified version number
```